README.md

```diff
@@ -144,6 +161,104 @@ module.exports = ({ renderJsBundles, renderCssLinks, production }) => {
 };
 ```
 
+## Deployment environments
+
+### Env vars
+
+Underreact allows you to inject environment variables **during the build time**. You can set them up by creating a `.env` file at the root of your project:
+
+```
+#.env
+SERVER=https://example.com
+```
+
+You can then use them in your codebase:
+
+```javascript
+const url = process.env.SERVER;
+fetch(`${url}/data`);
+```
+
+Any env variable that you use in your app must be declared in the `.env` file. This is required to prevent unintentional leaking of env vars in your javascript bundle. 
```

Comment:
Could we add another clause to further clarify: "All the variables in your environment will *not* be automatically available in your client-side code: any environment variable that you want to use in your app must be declared in the `.env` file."

And that second sentence might be concerning to someone who doesn't know the details. Maybe we should add a sentence explaining that only the environment variables will end up in your client-side bundle, and therefore loaded by browsers. So, of course, you should be discrete about which environment variables you use.